docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,347 @@
+
+* student_view_data: (list) Indicates for which block types the JSON data is
+  returned in the "student_view_data" response value. Example:
+  ``student_view_data=video``.
+
+* username: (string) Required, unless ``all_blocks`` is specified. Example:
+  ``username=anjali``.
+
+
+=====================
+Response Values
+=====================
+
+The following fields are returned with a successful response.
+
+* root: The ID of the root node of the requested course block structure.
+
+* blocks: A dictionary that maps block usage IDs to a collection of information
+  about each block. Each block contains the following fields.
+
+  * id: (string) The usage ID of the block.
+
+  * type: (string) The type of block. Possible values include course, chapter,
+    sequential, vertical, html, problem, video, and discussion. The type can
+    also be the name of a custom type of block.
+
+  * display_name: (string) The display name of the block.
+
+  * children: (list) If the block has child blocks, a list of IDs of the child
+    blocks. Returned only if "children" is included in the "requested_fields"
+    parameter.
+
+  * block_counts: (dict) For each block type specified in the block_counts
+    parameter to the endpoint, the aggregate number of blocks of that type for
+    this block and all of its descendants. Returned only if the "block_counts"
+    input parameter contains this block's type.
+
+  * graded (boolean) Whether or not the block or any of its descendants is
+    graded. Returned only if "graded" is included in the "requested_fields"
+    parameter.
+
+  * format: (string) The assignment type of the block. Possible values can be
+    "Homework", "Lab", "Midterm Exam", and "Final Exam". Returned only if
+    "format" is included in the "requested_fields" parameter.
+
+  * student_view_data: (dict) The JSON data for this block. Returned only if the
+    "student_view_data" input parameter contains this block's type.
+
+  * student_view_url: (string) The URL to retrieve the HTML rendering of this
+    block's student view. The HTML can include CSS and Javascript code. This
+    field can be used in combination with the ``student_view_multi_device``
+    field to decide whether to display this content to the user. This URL can
+    be used as a fallback if the ``student_view_data`` for this block type is
+    not supported by the client or by the block.
+
+  * student_view_multi_device: (boolean) Whether or not the block's rendering
+    obtained via block_url has support for multiple devices. Returned only if
+    "student_view_multi_device" is included in the "requested_fields"
+    parameter.
+
+  * lms_web_url: (string) The URL to the navigational container of the xBlock
+    on the web LMS. This URL can be used as a further fallback if the
+    student_view_url and the student_view_data fields are not supported.
+
+  * lti_url: The block URL for an LTI consumer.
+
+
+================
+Return Values
```

Comment:
These are HTTP status codes. I think "return values" sounds too much like "Response Values" above. 

I don't think we need to enumerate the possible HTTP status codes that the LMS can return. I know that in my quick testing, I saw more status codes than this and it would be exhausting to provide all the possible reasons for a 400 bad request. I suggest omitting all the "Return Values" sections if they are only listing HTTP status codes.